src/components/Authentication/Signup/Signup.jsx

```diff
@@ -0,0 +1,141 @@
+import loginSignupImage from "../../../assets/login-signup.png";
+import "@fontsource/dm-sans";
+import { useFormik } from "formik";
+import * as Yup from "yup";
```

Comment:
for performance reasons we can import only the stuff that we need from yup like: 

import {object, shape, string } from "yup"

Reply:
Fixed.